linera-service/src/cli_wrappers/remote_net.rs

```diff
@@ -20,14 +20,19 @@ pub struct RemoteNetTestingConfig {
 }
 
 impl RemoteNetTestingConfig {
+    /// Creates a new [`RemoteNetTestingConfig`] for running tests with an external Linera
+    /// network.
+    ///
+    /// The `faucet_url` is used to connect to the network and obtain its configuration,
+    /// as well as to create microchains used for testing. If the parameter is [`None`],
+    /// then it falls back to the URL specified in the `LINERA_FAUCET_URL` environment
+    /// variable, or the default devnet faucet URL.
     pub fn new(faucet_url: Option<String>) -> Self {
         Self {
-            faucet: Faucet::new(
-                faucet_url.unwrap_or(
-                    env::var("LINERA_FAUCET_URL")
-                        .unwrap_or("https://faucet.devnet.linera.net".to_string()),
-                ),
-            ),
+            faucet: Faucet::new(faucet_url.unwrap_or_else(|| {
+                env::var("LINERA_FAUCET_URL")
+                    .unwrap_or_else(|_| "https://faucet.devnet.linera.net".to_owned())
+            })),
```

Comment:
Maybe a bit easier on the eye:

```suggestion
            faucet: Faucet::new(faucet_url
                .or_else(|| env::var("LINERA_FAUCET_URL"))
                .unwrap_or_else(|_| "https://faucet.devnet.linera.net".to_owned())
            )),
```

Reply:
Good idea, thanks!

Reply:
Oh wait, one is a `Result` and the other is an `Option`. I've fixed it using `.ok()` :+1: